stable/ambassador/values.yaml

```diff
@@ -184,5 +214,5 @@ pro:
     secret: false
 
 crds:
-  create: true
+  create: false
```

Comment:
Is this intentional?

Reply:
This would definitely be a breaking change at least :D

Reply:
That was not. Good catch 😄